src/lib/controllers/ui-controller.js

```diff
@@ -526,7 +526,9 @@ export default class UIController {
 
         setInflData (state, hasInflData = true) {
           state.inflectionsWaitState = false
-          state.hasInflData = hasInflData
+          state.hasInflData = hasInflData &&
```

Comment:
@kirlat, it seems to me that setInflData and resetInflData - do absolutely the same thing with different hasInflData value.
Do we need them both?
Also is it consistent to check availability of getting inflections in setting inflection data flag? May be it is proper to check it before setting Vuex properties?
It is only my opinion, but it could be a little confusing - anyway in the code someone sets true to flags, but they won't be set to true without any errors in console.

Reply:
`resetInflData` always resets data to its default values, `setInflData` can set `hasInflData` to either `true` or `false`. It is consistent with other cases (we have `resetGrammarRes`) so I think it's probably better to keep it this way (at least for this build, this one is dedicated to performance optimization).

I agree that it's better to take availability check out of a mutation. I will put it to a `onHomonymReady()` callback. `hasInflData` is set to `true` there, if we do have some inflection data.